hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/BaseProcedure.scala

```diff
@@ -50,7 +50,7 @@ abstract class BaseProcedure extends Procedure {
     HoodieWriteConfig.newBuilder
       .withPath(basePath)
       .withIndexConfig(HoodieIndexConfig.newBuilder.withIndexType(IndexType.BLOOM).build)
-      .withRollbackUsingMarkers(false)
+      .withRollbackUsingMarkers(HoodieWriteConfig.ROLLBACK_USING_MARKERS_ENABLE.defaultValue.toBoolean)
```

Comment:
does not look like the right place to change. Looks like the original code is trying to overwrite and never use marker to rollback. otherwise it'll just skip setting it. This is for procedure maybe @XuQianJin-Stars can help clarify if we always want this to be `false`?